.github/workflows/ci.yaml

```
@@ -8,67 +8,72 @@
-    if: github.event.pull_request.head.repo.full_name == github.repository
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
+    # Check linters using ruff
+      - name: Check linters
+        run: uvx ruff check py_launch_blueprint/
+      - name: Upload coverage report to Codecov
+        uses: codecov/codecov-action@v4
+
+ # Only run security checks on pull requests from the main repository
+security:
```

actionlint (1.7.4)

unexpected key "security" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)